adoc/admin-crio-proxy.adoc

```diff
@@ -1,13 +1,12 @@
 == Configuring HTTP/HTTPS Proxy for {crio}
 
 In some cases you must configure the container runtime to use a proxy to pull
-container images. To configure this for {crio} you must modify the file
-`/etc/sysconfig/crio`.
+container images. The {crio} runtime uses the system proxy configuration,
+defined at `/etc/sysconfig/proxy`. This file can be edited by hand, or via
+YaST. For detailed instructions, see <insert link to SLES 15.1 docs and maybe
+that knowledge base item here>.
```

Comment:
I don't know what to do about a link in here.  There are a couple of potential link targets in the PR opening comment, but I don't know what the expected mechanism might be for linking back to SLES docs.  Or if we should do that at all.  Help, @r0ckarong :)

Reply:
If you have multiple links, just add multiple links as a bulleted list. Just use the link to SLE 15 SP1 docs as you find them on documentation.suse.com. Right now we do not have a good mechanism to cross reference documents that are not "our own" (anything outside CaaSP). External linking is fine. If this gets updated in the future the link will be adjusted.